hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -975,6 +975,13 @@ class HoodieSparkSqlWriterInternal {
       val metaSyncSuccess = metaSync(spark, HoodieWriterUtils.convertMapToHoodieConfig(parameters),
         tableInstantInfo.basePath, schema)
 
+      if (metaSyncSuccess) {
+        log.info("metaSyncSuccess " + tableInstantInfo.instantTime + " successful!")
+      }
```

Comment:
Kind of remember if the meta sync failes, the whole job would just fail.

Reply:
problem is job would stop with out any error or details to dig cause，these would take more work without the log @danny0405